static/docs/commands-reference/add.md

```diff
@@ -134,12 +134,10 @@ $ tree
 Let's check the `data.xml.dvc` file inside:
 
 ```yaml
-md5: aae37d74224b05178153acd94e15956b
+md5: 972537cc409b58e29ffc5c0a779f8dcf
```

Comment:
where do you take these updated md5s?

Reply:
just updated, copied the output of actual `.dvc` file.

Reply:
k, but where did you take the data.xml in the first place? :) this example does not specify the location. If we really want to change this (and we should update some command reference examples eventually), let's use the data.xml from the get started I would say.

Reply:
sure.